packages/cdktf/lib/manifest.ts

```diff
@@ -4,6 +4,7 @@ import * as path from "path";
 import * as fs from "fs";
 import { TerraformStack } from "./terraform-stack";
 import { AnnotationMetadataEntryType } from "./annotations";
+import stringify = require("json-stable-stringify");
```

Comment:
Nit: does this need to be a require import or can it be a normal static one?

Reply:
I think it has to be. I tried the other one first, but the LSP told me not to do that.

Reply:
ok :) I didn't yet grasp the different import and export styles fully, but if TS is ok with it I won't object :)